Makefile

```diff
@@ -471,6 +482,11 @@ $(BINDIR)/$(UNIT_DIR)/size_str-test: INCLUDE += -I ./$(L3_UTILSDIR)
 $(BINDIR)/$(UNIT_DIR)/size_str-test: $(OBJDIR)/$(UNITTESTS_DIR)/size_str-test.o \
                                      $(OBJDIR)/$(L3_UTILSDIR)/size_str.o
 
+# Unit-test for l3-write interfaces needs this defined so that l3_log()
+# gets correctly redefined to l3_log_fprintf() / l3_log_write() interface
+$(BINDIR)/$(UNIT_DIR)/l3-fprintf-perf-test: DFLAGS_UNIT := -DL3_LOGT_FPRINTF
+$(BINDIR)/$(UNIT_DIR)/l3-write-perf-test: DFLAGS_UNIT := -DL3_LOGT_WRITE
```

Comment:
The -D flags needed for conditional compilation are added here, only for the appropriate `make` target.